src/forks.rs

```diff
@@ -0,0 +1,238 @@
+use crate::bank_delta::BankDelta;
+/// This module tracks the forks in the bank
+use crate::bank_fork::BankFork;
+use std::sync::Arc;
+//TODO: own module error
```

Comment:
That's a good suggestion. Can you add that in this PR to avoid introducing that bank dependency?